TheBTeam.ConsoleApp/MainMenu.cs

```diff
@@ -97,20 +118,34 @@ public static void ShowMainMenu()
                 else if (mainMenuItem[currentItem] == ("Show all transaction"))
                 {
                     Console.WriteLine($"{mainMenuItem[currentItem]}");
-                    TransactionViewer.ViewTransaction(tmpListTransactions.TransactionsList);
-                    Console.ReadKey();
+                    if (tmpListTransactions.TransactionsList.Count != 0)
+                    {
+                        TransactionViewer.ViewTransaction(tmpListTransactions.TransactionsList);
+                        Console.ReadKey();
+                    }
+                    else
+                    {
+                        Console.WriteLine($"No transactions, Please enter transaction");
+                        Console.WriteLine($"Press any key to continue");
+                        Console.ReadKey();
+                    };
                 }
                 else if (mainMenuItem[currentItem] == ("Show transaction according Category"))
                 {
                     Console.WriteLine($"{mainMenuItem[currentItem]}");
-                    TransactionViewer.ViewTransactionAccordingCategory(tmpListTransactions.TransactionsList);
-                    Console.ReadKey();
+                    if (tmpListTransactions.TransactionsList.Count != 0)
+                    {
+                        TransactionViewer.ViewTransactionAccordingCategory(tmpListTransactions.TransactionsList);
+                        Console.ReadKey();
```

Comment:
move it to methods! guess it will be cleaner and more practical

Reply:
about end

Reply:
I shifted it to transaction viewer from main menu